src/Programs/Programs.ts

```diff
@@ -222,9 +222,34 @@ export const Programs: Record<CompletedProgramName, Program> = {
         return;
       }
 
-      Terminal.print(targetServer.hostname + ":");
+      if (targetServer.hasAdminRights) {
+        Terminal.success("Server has admin rights.");
+        Terminal.success(targetServer.hostname + ":");
+      } else {
+        Terminal.error("Server has no admin rights.");
+        Terminal.error(targetServer.hostname + ":");
+      }
       Terminal.print("Server base security level: " + targetServer.baseDifficulty);
-      Terminal.print("Server current security level: " + targetServer.hackDifficulty);
+      if (targetServer.hackDifficulty > targetServer.baseDifficulty) {
```

Comment:
This is complicated and I'm not sure that it's a good idea. Let's revert this part for now, and do a different PR later that focuses on augmenting ServerProfiler if you still want to later. (The built-in.exe tools are *intentionally* crude to nudge you towards writing your own.)